masterlist.yaml

```
@@ -5287,11 +5287,10 @@ plugins:
 
   - name: 'ogFalkreathHauntings.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/20733/' ]
-    dirty:
-      - <<: *quickClean
-        crc: 0xF302A166
-        util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 3
+    msg:
+      - type: say
+        content: 'Do not clean. "Dirty" edits are intentional and required for the mod to function.'
+        lang: eng
```

You could use the alias doNotClean instead, it does have a slightly different message.

> Do not clean ITM records, they are intentional and required for the mod to function. It is safe to undelete records, but do not do anything other than that.

```suggestion
    msg: [ *doNotClean ]
```

---

That works for me if it works for you.